### PR TITLE
Number of notification produced has changed after the connection re-ordering was fixed

### PR DESCRIPTION
**Pull request by typfel**
depends on 
- [x] https://github.com/wireapp/wire-ios-mocktransport/pull/31

### CODE REVIEW

File: Tests/Source/Integration/ConversationsTests.m [hunk 1]
```diff
@@ -1162,7 +1162,7 @@ - (void)testThatConversationGoesOnTopAfterARemoteUserAcceptsOurConnectionRequest
         XCTAssertEqual(note.deletedIndexes.count, 0u);
         XCTAssertEqual(note.insertedIndexes.count, 0u);
     }
-    XCTAssertEqual(updatesCount, 1);
+    XCTAssertGreaterThanOrEqual(updatesCount, 1);
```
Comment (typfel) [inline Tests/Source/Integration/ConversationsTests.m]: This is flakey. See line: 1153

Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Can we get the diff before the action and after and assert on that?

Comment (typfel) [inline Tests/Source/Integration/ConversationsTests.m]: How do you mean? If don't care about the number of notification updates we can also remove this line.

Comment (daehn) [inline Tests/Source/Integration/ConversationsTests.m]: Yeah, I misunderstood what we're doing there

**Pull request merged**